files/10-kubeadm.conf

```diff
@@ -0,0 +1,11 @@
+# Note: This dropin only works with kubeadm and kubelet v1.11+
```

Comment:
please add a comment like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
done